clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -231,17 +248,23 @@ class GCEUtil {
                                                            Compute compute,
                                                            Task task,
                                                            String phase,
-                                                           SafeRetry safeRetry) {
+                                                           SafeRetry safeRetry,
+                                                           GoogleExecutorTraits executor) {
     task.updateStatus phase, "Looking up regional load balancers $forwardingRuleNames..."
 
     def forwardingRules = safeRetry.doRetry(
-      { return compute.forwardingRules().list(projectName, region).execute().items },
-      "list",
+      { return executor.timeExecute(
+            compute.forwardingRules().list(projectName, region),
```

Comment:
This is indented 4 spaces, but the one just above in the diff is only indented 2 spaces. Is that intentional?